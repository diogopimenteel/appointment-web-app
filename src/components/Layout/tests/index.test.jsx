/* eslint-disable react/react-in-jsx-scope */
/* eslint-disable no-undef */
import ReactDOM from 'react-dom';

import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom';

import renderer from 'react-test-renderer';
import Page from '../index';

afterEach(cleanup);

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Page />, div);
});

it('renders page correctly', () => {
  const { getByTestId } = render(<Page title="Test" />);
  expect(getByTestId('page-test')).toHaveTextContent('Test');
});

it('matches snapshot', () => {
  const tree = renderer.create(<Page title="Test" />).toJSON();
  expect(tree).toMatchSnapshot();
});
