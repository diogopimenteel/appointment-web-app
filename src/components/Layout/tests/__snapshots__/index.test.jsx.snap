// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="mt-4 container"
  data-testid="layout-test"
>
  <div
    align="center"
  >
    <div
      className="card text-center card"
      data-testid="card-test"
    >
      <div
        className="card-header"
      >
        <div
          className="card-title h5"
        >
          Test
        </div>
      </div>
      <div
        className="card-body"
      />
    </div>
  </div>
</div>
`;
